scipy/linalg/tests/test_basic.py

```diff
@@ -1089,13 +1090,13 @@ def test_1x1_all_singleton_dims(self):
         a = np.array([[[[1]]]], dtype='f')
         deta = det(a)
         assert deta.dtype.char == 'd'
-        assert np.isscalar(deta)
```

Comment:
why remove the scalar check?

Reply:
Because the result is not a scalar. It is an array with shape (1, 1).

Reply:
ah of course :man_facepalming: makes sense!